### ISSUE TITLE
Small borders

### ISSUE THREAD

**Issue by piranna** [chjj/blessed#126]
Add a new border style besides line and bg that uses [block element](http://en.wikipedia.org/wiki/Block_Elements) chars. With the new dockBorders functionality, it would use the background of one cell and the background of the other to fill the two halfes, and also would show a smaller shadow. This can be useful for interfaces with panels to have a place where to resize them.

**Comment by chjj** [comment]
Yeah, this should be easy to implement since we already have ACS borders. The docking part might be trickier. There's no block elements like: '┬', '┼', etc. So the effect of '┬' would have to be two '┐''┌' next to each other since they appear to side on the edge of the cell, which wouldn't actually require dockBorders.

**Comment by piranna** [comment]
Definitelly didn't though about the corners, there would be problems when combining two corners and a background or for corners since they would need up to three or four diferent colors, but for only horizontal or vertical borders is feasable.

**Comment by piranna** [comment]
Also, I think this colors problem would also arise with the regular docking borders if both windows has diferent colors... :-/

**Comment by chjj** [comment]
@piranna, yeah, I had to tackle that problem when implementing dockBorders. Elements with 2 different borders colors do not dock borders because it looks extra funky. You get a multicolored border.

**Comment by piranna** [comment]
Ah, ok, cool :-)
